docs/source/schemas/tables_and_indices.rst

```diff
@@ -213,6 +215,24 @@ as descriptive as is useful. If this method is not called, the value
 will be derived by converting the table's AtlasDB name from snake\_case
 to CamelCase.
 
+.. code:: java
+
+    public void tableNameIsSafeLoggable();
+
+If called, this indicates that the table name is safe for logging.
+Table names are considered unsafe by default.
+
+.. code:: java
+
+    public void namedComponentsSafeByDefault();
+
+If called, then row components and named columns that are subsequently
+defined for this table will be assumed to be safe for logging unless
+specifically indicated as unsafe. By default, row components and named
+columns are assumed unsafe unless specifically indicated as safe.
+
+Note that the ambit of this method does not extend to the table name itself.
```

Comment:
This sentence is a bit opaque, especially for non-native speakers. Spell it out a bit more. Something like: "Note that specifying named components as safe by default does not also make the table name considered safe".